### DISCUSSION TITLE
Spot instances per process in GCP

### DISCUSSION THREAD

**Discussion by mizue31g** [nextflow-io/nextflow#5252] [Q&A]
Hi Nextflow team,
I'm planning to use cloud-batch executors with Nextflow. Is it possible to specify a spot instance for each process? I want to use non-spot instances for some processes. 
Looking at the following issue, it seems that you can't specify the machine type for flexible, so I'm asking this question.
https://github.com/nextflow-io/nextflow/issues/4657

Thanks,

**Comment by bentsherman** [comment]
It is not possible, you can only enable/disable spot for all processes

**Comment by gluonfield** [comment]
Are there any progress on this issue? It seems quite critical to be able to run some tasks on spot instances to save cost, while being able to run long running processes with onDemand machines.

**Comment by stianlagstad** [comment]
We would also like to see this feature. We would use spot instances for shorter jobs, but normal instances for long-running jobs that would be expensive to start over (such as dna and rna alignment).
